sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -333,6 +333,13 @@ object SQLConf {
     .booleanConf
     .createWithDefault(true)
 
+  val EXTENDED_EXPLAIN_PROVIDER = buildConf("spark.sql.extendedExplainProvider")
```

Comment:
`StaticSQLConf` might be a better place for this conf. We don't want to change this per query, right?